ui/src/components/Layout/Aside/Aside.tsx

```diff
@@ -95,15 +97,10 @@ export const Aside = () => {
               </Hug>
             </AsideButton>
           </Hug>
-          {/* <AsideButton active={active === 1} onClick={() => setActive(1)}>
-            <Hug as="span" ml={4}>
-              <DNA />
-            </Hug>
-            <Hug as="span" ml={16}>
-              Something else
-            </Hug>
-          </AsideButton> */}
-        </div> {/* the thing that holds buttons  */}
+        </Hug>
+        <Hug mt="auto" mb={16}>
+          <a href="https://foxops.readthedocs.io/" target="_blank" rel="noreferrer">Docs</a>
```

Comment:
Can we also get an addition link for the `API docs` ? basically redirecting to `/docs` ? It can also be in an additional PR :)